index.html

```diff
@@ -2704,40 +2704,36 @@ <h3>Processing Errors</h3>
 below.
           </li>
           <li>
-The `code` value MUST be the integer code described in the table below
-(in parentheses, beside the type name).
-          </li>
-          <li>
-The `title` value SHOULD provide a short but specific human-readable string for
+The `title` value MUST be present and it SHOULD provide a short but specific human-readable string for
```

Comment:
This should match what's in the other specs and not add a new MUST: https://w3c.github.io/vc-bitstring-status-list/#processing-errors

```suggestion
The `title` value SHOULD provide a short but specific human-readable string for
```

Reply:
See https://github.com/w3c/vc-data-integrity/pull/327#discussion_r1911962360